ticketbuyer/buyer.go

```diff
@@ -323,13 +302,26 @@ func (t *TicketPurchaser) Purchase(height int64) (*PurchaseStats, error) {
 	}
 	ps.PriceNext = sDiffEsts.Expected
 
+	// Set the max price to the configuration parameter that is lower
+	// Absolute or relative max price
+	var maxPriceAmt dcrutil.Amount
+	if t.cfg.MaxPriceAbsolute > 0 && t.cfg.MaxPriceAbsolute < avgPrice*t.cfg.MaxPriceRelative {
+		maxPriceAmt, err = dcrutil.NewAmount(t.cfg.MaxPriceAbsolute)
+		if err != nil {
+			return ps, err
+		}
+		log.Debugf("Using absolute max price of %v", maxPriceAmt)
+	} else {
+		maxPriceAmt, err = dcrutil.NewAmount(avgPrice * t.cfg.MaxPriceRelative)
+		if err != nil {
+			return ps, err
+		}
+		log.Debugf("Using relative max price of %v", maxPriceAmt)
+	}
+
```

Comment:
This check is done in reference to the calculated avgPrice can't be done in loadConfig, the check is updated each new block connected.